tests/integration/local/start_lambda/test_start_lambda.py

```diff
@@ -0,0 +1,164 @@
+
+    def setUp(self):
+        self.url = "http://127.0.0.1:{}".format(self.port)
+        self.lambda_client = boto3.client('lambda',
+                                          endpoint_url=self.url,
+                                          use_ssl=False,
+                                          verify=False,
+                                          config=Config(signature_version=UNSIGNED,
+                                                        read_timeout=120,
+                                                        retries={'max_attempts': 0}))
+
+    def test_same_endpoint(self):
+        """
+        Send two requests to the same path at the same time. This is to ensure we can handle
+        multiple requests at once and do not block/queue up requests
+        """
+        number_of_requests = 10
+        start_time = time()
+        thread_pool = ThreadPoolExecutor(number_of_requests)
+
+        futures = [thread_pool.submit(self.lambda_client.invoke, FunctionName="HelloWorldSleepFunction")
+                   for _ in range(0, number_of_requests)]
+        results = [r.result() for r in as_completed(futures)]
+
+        end_time = time()
+
+        self.assertEquals(len(results), 10)
+        self.assertGreater(end_time - start_time, 10)
+        self.assertLess(end_time - start_time, 20)
+
+        for result in results:
+            self.assertEquals(result.get("Payload").read().decode('utf-8'), '"Slept for 10s"')
+
+
+class TestLambdaToLambdaInvoke(StartLambdaIntegBaseClass):
+    template_path = "/testdata/start_lambda/template.yml"
+
+    def setUp(self):
+        self.url = "http://127.0.0.1:{}".format(self.port)
+        self.lambda_client = boto3.client('lambda',
+                                          endpoint_url=self.url,
+                                          use_ssl=False,
+                                          verify=False,
+                                          config=Config(signature_version=UNSIGNED,
+                                                        read_timeout=120,
+                                                        retries={'max_attempts': 0}))
+
+    def test_local_lambda_calling_local_lambda(self):
+        pass
+
+
+class TestLambdaServiceErrorCases(StartLambdaIntegBaseClass):
+    template_path = "/testdata/invoke/template.yml"
+
+    def setUp(self):
+        self.url = "http://127.0.0.1:{}".format(self.port)
+        self.lambda_client = boto3.client('lambda',
+                                          endpoint_url=self.url,
+                                          use_ssl=False,
+                                          verify=False,
+                                          config=Config(signature_version=UNSIGNED,
+                                                        read_timeout=120,
+                                                        retries={'max_attempts': 0}))
+
+    def test_invoke_with_non_json_data(self):
+        if sys.version_info.major == 2:
```

Comment:
Error message from the CLI should not differ between Python versions. LOG.debug() the actual exception message and remove it from the response msg here